assets/app/styles/_components.less

```diff
@@ -47,6 +47,10 @@
     border: 1px solid #ddd;
     position: relative;
     .osc-object-highlight();
+    h2 {
+      // Avoid clipping descenders in routes on overview.
+      line-height: 1.3;
```

Comment:
surprised the line-height for the h2 wasnt correct, was something overriding the base styles?  maybe we need to fix our _variables.less file if not